sdk/test/UnitTests/TestLambdaContext.cs

```diff
@@ -48,6 +55,53 @@ public new void TestCleanup()
             Environment.SetEnvironmentVariable(AWSXRayRecorder.LambdaTraceHeaderKey, null);
         }
 
+        [TestMethod]
+        public void TestLamdaSampled()
+        {
+            var mockEmitter = new Mock<ISegmentEmitter>();
+
+            using (AWSXRayRecorder recorder = AWSXRayRecorderFactory.CreateAWSXRayRecorder(mockEmitter.Object))
+            {
+                AWSXRayRecorder.InitializeInstance(recorder: _recorder);
+
+                LambdaTestHelper(recorder, true);
+                mockEmitter.Verify(x => x.Send(It.IsAny<Subsegment>()), Times.Exactly(2));
+
+                LambdaTestHelper(recorder, false);
+                mockEmitter.Verify(x => x.Send(It.IsAny<Subsegment>()), Times.Exactly(2));
+
+                LambdaTestHelper(recorder, true);
+                mockEmitter.Verify(x => x.Send(It.IsAny<Subsegment>()), Times.Exactly(4));
+
+                LambdaTestHelper(recorder, false);
+                mockEmitter.Verify(x => x.Send(It.IsAny<Subsegment>()), Times.Exactly(4));
+            }
+        }
+
+        private static void LambdaTestHelper(AWSXRayRecorder recorder, bool sampled)
```

Comment:
I think this test should be renamed to follow the naming pattern of the other tests. e.g: `TestLambdaEnv` or `TestLambdaHelper`

Reply:
Done